cypress/page/address-step.page.ts

```diff
@@ -0,0 +1,12 @@
+class AddressPage {
+    private checkoutBtn: string;
+
+    constructor() {
+      this.checkoutBtn = ".cart_navigation button";
+    }
+
+    public checkoutBtnEvent(): void {
```

Comment:
the methods on page objects generally should be called with the name of business actions, without referring for example to the element or events like in this case BtnEvent... instead of that, you can call it like confirmAddress or selectDefaultAddress or something similar